roles/ovirt-vm-infra/README.md

```diff
@@ -87,13 +89,25 @@ Example Playbook
       memory: 2GiB
       cores: 2
 
+    critical_vm:
```

Comment:
Please just add the storage_domain parameter to httpd_vm profile, I think it's not needed to create just another profile because of that. Let's keep it simple.